Gujarati/અમે_ગુજરાતી/અમે_ગુજરાતી.txt

```diff
@@ -0,0 +1,2 @@
+koi pan jagya hoy guajrati game tya pochi jay 
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.